wannierberri/system/sym_wann.py

```diff
@@ -309,26 +309,21 @@ def atom_rot_map(self, sym):
         rot_sym = self.symmetry['rotations'][sym]
         rot_sym_glb = np.dot(np.dot(np.transpose(self.lattice), rot_sym), np.linalg.inv(np.transpose(self.lattice)))
         if self.soc:
+            sym_only = True
+            sym_T = True
             if self.magmom is not None:
                 for i in range(self.num_wann_atom):
-                    magmom = np.round(self.wann_atom_info[i][-2], decimals=4)
-                    new_magmom = np.round(np.dot(rot_sym_glb, magmom), decimals=4)
-                    if abs(np.linalg.norm(magmom - np.linalg.det(rot_sym_glb) * new_magmom)) > 0.0005:
-                        sym_only = False
-                    else:
-                        sym_only = True
-                        print('Symmetry operator {} respect magnetic moment'.format(sym + 1))
-                    if abs(np.linalg.norm(magmom + np.linalg.det(rot_sym_glb) * new_magmom)) > 0.0005:
-                        sym_T = False
-                    else:
-                        sym_T = True
-                        print('Symmetry operator {}*T respect magnetic moment'.format(sym + 1))
-                    if sym_T + sym_only == 0:
-                        break
-
-            else:
-                sym_only = True
-                sym_T = True
+                    if sym_only or sym_T:
+                        magmom = np.round(self.wann_atom_info[i][-2], decimals=4)
+                        new_magmom = np.round(np.dot(rot_sym_glb, magmom), decimals=4)*np.linalg.det(rot_sym_glb)
```

Comment:
Do we really need round here?

Reply:
Yes, it is not needed. (if we don't copy the result from VASP directly. And better not copy that).